dashboards/ansible-automation-controller/dashboard.json

```diff
@@ -231,9 +247,9 @@
 				{
 					"title": "Organizations, Teams and Users",
```

Comment:
@pkudikyala,
In this chart query you mentioned Organisations so change to "Organizations"

**Suggest:**
"query": "FROM Metric SELECT latest(awx_organizations_total) AS 'Organizations', latest(awx_teams_total) AS 'Teams', latest(awx_users_total) AS 'Users' "

Reply:
Hi @mdumpati ,
As per your comment, I have changed the card title. Kindly review it.

Thank you...